omod/src/main/webapp/resources/htmlforms/122a-HIVCare_ARTCard-EncounterPage.xml

```diff
@@ -514,6 +539,16 @@
 
                         </excludeIf>
 
+                        <span class="OtherAppointmentReasonEnableDisable1">
```

Comment:
@slubwama Why is the class OtherAppointmentReasonEnableDisable1 - should it not be named ReasonForNextAppointmentEnableDisable1? As the field does not exist anywhere else.